src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionHelpers.cs

```diff
@@ -325,26 +382,34 @@ private static CodeAction GetNestedActionsFromActionSet(IUnifiedSuggestedAction
             {
                 foreach (var action in actionSet.Actions)
                 {
-                    nestedActions.Add(GetNestedActionsFromActionSet(action));
+                    nestedActions.Add(GetNestedActionsFromActionSet(action, fixAllScope));
+                    if (fixAllScope != null)
+                    {
+                        var fixAllAction = GetFixAllActionsFromActionSet(action, fixAllScope);
```

Comment:
think it would be cleaner if you just passed the arraybuilder into `GetFixAllActionsFromActionSet` and moved all this logic from here (and from line 355) into that method.  less duplication